src/constants/query-keys.ts

```diff
@@ -0,0 +1,3 @@
+export enum QUERY_KEYS {
```

Comment:
er bestaat al een `shared/const` maar als dit specifiek voor de reading rooms is mag je dat ook plaatsen onder `reading-room/const`